builder/tart/step_run.go

```diff
@@ -207,6 +207,12 @@ func typeBootCommandOverVNC(
 
 	ui.Say("Connected to the VNC!")
 
+	if config.BootWait != 0 {
+		message := fmt.Sprintf("Waiting %v for VM to boot...", config.BootWait)
```

Comment:
Just some nit picks about wording. I think this phrasing is a bit more accurate.

```suggestion
		message := fmt.Sprintf("Waiting %v after the VM has booted...", config.BootWait)
```